Jenkinsfile

```diff
@@ -203,7 +203,9 @@ VERSION=${env.VERSION}-SNAPSHOT""")
       dir("${BASE_DIR}"){
         notifyBuildResult(prComment: true,
                           slackComment: true,
-                          analyzeFlakey: !isTag(), jobName: getFlakyJobName(withBranch: getFlakyBranch()))
+                          analyzeFlakey: !isTag(), jobName: getFlakyJobName(withBranch: getFlakyBranch()),
```

Comment:
`analyzeFlakey` it seems to have been misspelled, worked anyway?

Reply:
Thanks for noting this, this change is not about the name of the `analyzeFlakey ` attribute but the `comma` at the very end to add the githubIssue notifications.

It's misspelled indeed, but didn't change it since consumers will also need to change it, and need to keep backward compatibility as we don't know who consumes it :/